tests/DoctrineAuditBundle/AuditConfigurationTest.php

```diff
@@ -51,6 +51,21 @@ public function testCustomTableSuffix(): void
         $this->assertSame('_audit_log', $configuration->getTableSuffix(), 'custom table_suffix is "_audit_log".');
     }
 
+    public function testDefaultEnabled(): void
+    {
+        $configuration = $this->getAuditConfiguration();
+
+        $this->assertTrue($configuration->getEnabled(), 'Enabled by default.');
+    }
+
+    public function testSetEnabledToFalse(): void
```

Comment:
You should also test the `enable` method.

Reply:
I'd have named these `testEnable` and `testDisable` to ease understanding/reading.